### DISCUSSION TITLE
How to config kong skywalking plugin

### DISCUSSION THREAD

**Discussion by 807801002** [apache/skywalking#7983] [Q&A]
I set up the configuration of kong according to https://github.com/apache/skywalking-kong/blob/master/README.md, but it response "{"message":"Not found"}" when I curl http://localhost:8001/plugins/ by following:
curl -X POST --url http://localhost:8001/plugins/ \
   --data 'name=skywalking' \
   --data 'config.backend_http_uri=http://localhost:12800' \
   --data 'config.sample_ratio=100' \
   --data 'config.service_name=kong' \
   --data 'config.service_instance_name=kong-with-skywalking'
And no data sent to skywalking-oap

**Comment by wu-sheng** [comment]
Check the Kong's version? Or at least check the logs. Otherwise, we can't help.

**Reply by 807801002** [reply]
kong's version is 2.5.0, no error log at error.log
